docs/application/dotnet/get-started/first-app.md

```diff
@@ -0,0 +1,495 @@
+        <button id="run-button3" class="tablinks" onclick="openTabSection(event, 'CLI-Run', 'TabSection3')">CLI</button>
+    </div>
+    <div id="Visual-Studio-2019-Run" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+                    <ol>
+                        <li>Launch <b>Tizen Emulator Manager</b> from menu <b>Tools > Tizen > Tizen Emulator Manager</b>.</li>
+                        <img alt=" " src="media/vs2019_run_1.png"/>
+                        <p></p>
+                        <li>Choose Emulator. For basic NUI template, there is no difference which platform is used. In this example, the application is tested on TV emulator.</li>
+                        <img alt=" " src="media/vs2019_run_2.png"/>
+                        <li>When emulator window is visible you can switch back to the Visual Studio.</li>
+                        <img alt=" " src="media/vs2019_run_3.png"/>
+                        <li>In Visual Studio, new launch mode should be visible with valid emulator name. To deploy and run application, green arrow can be used.</li>
+                        <img alt=" " src="media/vs2019_run_4.png"/>
+                        <li>Finally, <b>HelloWorld</b> project UI is visible on emulator screen.</li>
+                        <img alt=" " src="media/vs2019_run_5.png"/>
+                        <p></p>
+                    </ol>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <div id="Visual-Studio-Code-Run" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+                    <ol>
+                        <li>Use <b>ctrl + P</b> to show command panel.</li>
+                        <li>Type <b>Tizen .NET: Emul</b> and launch the Tizen Emulator Manager.</li>
+                        <img alt=" " src="media/vscode_run_1.png"/>
+                        <p></p>
+                        <li>Choose Emulator. For basic NUI template, there is no difference which platform is used. In this example, the application is tested on TV emulator.</li>
+                        <img alt=" " src="media/vs2019_run_2.png"/>
+                        <p></p>
+                        <li>When emulator window is visible, you can switch back to the Visual Studio Code.</li>
+                        <img alt=" " src="media/vs2019_run_3.png"/>
+                        <li>Use <b>ctrl + P</b> to open command panel again and type <b>Tizen .NET: Set</b> and set the Tizen Device.</li>
+                        <img alt=" " src="media/vscode_run_2.png"/>
+                        <p></p>
+                        <p>A popup in the right corner of the Visual Studio code appears.</p>
+                        <img alt=" " src="media/vscode_run_3.png"/>
+                        <p></p>
+                        <li>Use <b>ctrl + P</b> to open the command panel and type <b>Tizen .NET: Run</b> and choose available option</li>.
```

Comment:
Global comment:

Please ensure periods (.) are included inside closing tags, so they display properly in the rendered view. If periods are included outside the closing tag, they wont appear in the rendered view.

```suggestion
                        <li>Use <b>ctrl + P</b> to open the command panel and type <b>Tizen .NET: Run</b> and choose available option.</li>
```